src/locales/en_US.ts

```diff
@@ -192,6 +192,7 @@ const strings = {
   fragment_transaction_list_tx_dropped: 'Dropped',
   fragment_transaction_list_tx_synchronizing: 'Synchronizing',
   fragment_transaction_list_confirmation_progress: '%s of %s Confirmations',
+  fragment_transaction_list_unconfirmed_rbf: 'Awaiting confirmation, may be replaced',
```

Comment:
I think Michael wants wording like "may be cancelled". We want normal people to understand that "you may not get the money". The word "cancelled" does this better than "replace".

Reply:
Description says cancelled, his comment says replaced. I agree cancelled is better so I changed it